static/scss/answers/common/variables.scss

```diff
@@ -9,8 +9,8 @@ $vertical-margin-top: $grid-gutter-width !default;
 $vertical-margin-bottom: $grid-gutter-width !default;
 
 $screen-xs-min: 480px;
-$screen-xs-max: 767px;
-$screen-sm-min: 768px;
+$screen-xs-max: 768px;
```

Comment:
did we mean to change screen-xs-max from 767 to 768?

Reply:
no, but it doesn't make a difference because we are no longer using those values in utilitymixins. I'll change it back though just for more clarity.

Reply:
no, but it doesn't make a difference because we are no longer using those values in utilitymixins. I'll change it back though just for more clarity.